android/java/org/chromium/chrome/browser/settings/AppearancePreferences.java

```diff
@@ -77,7 +77,7 @@ public void onActivityCreated(Bundle savedInstanceState) {
         if (hideBraveRewardsIconPref != null) {
             SharedPreferences sharedPreferences = ContextUtils.getAppSharedPreferences();
             hideBraveRewardsIconPref.setChecked(
-                    sharedPreferences.getBoolean(PREF_HIDE_BRAVE_REWARDS_ICON, false));
+                    !sharedPreferences.getBoolean(PREF_HIDE_BRAVE_REWARDS_ICON, false));
```

Comment:
I was a little confused by this line, but I'm not familiar with the changes required on Android.

Reply:
Yeah, it is confusing. On Android, the preference for this setting is completely separate from the related brave-core preference. I chatted with @deeppandya and we thought the best way forward is to keep the pref as-is for now on Android (so just reverse the logic to match the new text for this toggle) and then create a new issue to update Android to use the brave-core preference, migrating any non-default values.

Open to other ideas too, though.